tests/foreman/api/test_remoteexecution.py

```diff
@@ -73,13 +73,15 @@ def test_positive_run_capsule_upgrade_playbook(module_capsule_configured, target
 @pytest.mark.tier3
 @pytest.mark.no_containers
 @pytest.mark.rhel_ver_list('8')
+@pytest.mark.parametrize('setting_update', ['remote_execution_global_proxy=False'], indirect=True)
```

Comment:
with unchanged parameter name, pytest has issues with `=False` in the test name (see your recent PRT job).
I'd suggest calling the parameter `no-rex-global-proxy` or so.
you can do it by passing another parameter, called `ids`
[relevant docs](https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids)